src/flat-button.jsx

```diff
@@ -197,7 +258,7 @@ const FlatButton = React.createClass({
         onKeyboardFocus={this._handleKeyboardFocus}
         onMouseLeave={this._handleMouseLeave}
         onMouseEnter={this._handleMouseEnter}
-        onTouchStart={this._handleTouchStart}
```

Comment:
Ok, I guess it makes sense to remove it since the only use it to pass it down to EnhancedButton and its not used there.